index.html

```diff
@@ -56,49 +53,38 @@ <h2>Introduction to EVM Object Format</h2>
 						<!-- First Section -->
 							<section id="first" class="main special">
 								<header class="major">
-									<h2>EOF EIPs</h2>
+									<h2>Why?</h2>
 								</header>
 								<ul class="features">
 									<li>
-										<span class="icon solid major style1 fa-check"></span>
-                    <h3><a href="https://eips.ethereum.org/EIPS/eip-3541">EIP-3541</a></h3>
-                    <p>Rejects deploying code starting with the byte 0xef. This EIP is already live on Ethereum Mainnet</p>
+                    <h3>Code validation at deploy time</h3>
+                    <p>Instead of having the intepreter validate instructions repeatedly at runtime, EVM bytecode is validated once and during contract creation, making for a faster, cheaper, and more secure execution.</p>
```

Comment:
```suggestion
                    <p>Instead of having the intepreter validate instructions repeatedly at runtime, EVM bytecode is validated once for its lifetime during contract creation, making for a faster, cheaper, and more secure execution.</p>
```

Reply:
I like dropping the "and" here, but I'm not sure about the "for its lifetime" introduced, I think it's clear enough with just the "once" word. Can someone post a second opinion on this addition?

Reply:
I'm fine either way, also an option: "once" => "only once"